src/map/Map.js

```diff
@@ -624,6 +627,16 @@ L.Map = L.Evented.extend({
 		this._container.scrollLeft = 0;
 	},
 
+	_onMoveEnd: function () {
+		if (L.Browser.any3d && this.options.transform3DLimit &&
+			(Math.abs(this._mapPane._leaflet_pos.x) >= this.options.transform3DLimit
+			 || Math.abs(this._mapPane._leaflet_pos.y) >= this.options.transform3DLimit)) {

```

Comment:
I'm thinking about making a `L.Bounds` with the max values for 3D transforms and just use something like `max3dTransforms.contains(this._getMapPanePos())`.